bat/ZC390CL.BAT

```diff
@@ -0,0 +1,31 @@
+rem ZC390CL translate CBL to MLC and assemble/LINK using z390 
```

Comment:
We have the bat script CBLCL.BAT that appears to perform the same function as this module. Should zCICS just change to use this module rather than create a new one??
Also, not sure how zCICS fits into BSAM and QSAM file processing which is the topic of this pull request.

Reply:
As per @dsh33782  advice, have deleted ZC390CL.BAT in commit 9fed788